src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const formattedQuery = query.toUpperCase();
```

Comment:
```suggestion
    const formattedQuery = query.toUpperCase().trim();
```
<img width="648" alt="image" src="https://user-images.githubusercontent.com/55802630/184601058-ce109c03-e41f-4645-8f36-cb5e859189ac.png">